packages/components/addon/components/hds/form/textarea/base.hbs

```diff
@@ -0,0 +1,2 @@
+{{! Notice: this is not the native HTML <textarea> but the Ember component <Textarea> }}
+<Textarea class={{this.classNames}} {{style width=@width}} ...attributes @value={{@value}} />
```

Comment:
Doing more digging-
1. the `class` attribute should come _after_ `...attributes` so that any new classes are appended to the end of the class name set

When I added `class="end"` to the `<Textarea::field>` invocation, what I got is not what I expected: 
![CleanShot 2022-06-21 at 13 50 10@2x](https://user-images.githubusercontent.com/4587451/174877118-0a06278f-ac24-4d36-979a-ff42617d5737.png)

I am hesitant to say this because I know it's extra work, but I don't think that Ember's Textarea works the way this PR assumes it works; I think we have to build our own.

Reply:
@MelSumner I replied also in the Slack thread. I don't think this is an issue, the order of the class names in the `class` attribute [doesn't matter](https://css-tricks.com/the-order-of-css-classes-in-html-doesnt-matter/).